### ISSUE TITLE
Add a template for a web app with no database

### ISSUE THREAD

**Issue by stephendolan** [luckyframework/lucky_cli#627]
It'd be nice to have an option for `lucky init.custom` that would build you a website without a database.

We've got a few examples in the Lucky website itself and LuckyDiff:
https://github.com/luckyframework/website
https://github.com/stephendolan/lucky_diff

**Comment by jwoertink** [comment]
I think I want to wait on this until we have a full solution with https://github.com/luckyframework/lucky/discussions/1812 

That may change how this feature gets added in
